.codegen/impl.go.tmpl

```diff
@@ -40,11 +40,7 @@ func (a *{{.Service.CamelName}}Impl) {{.PascalName}}(ctx context.Context{{if .Re
 
 {{ define "response-var" -}}
 {{ if .Response }}
-  var {{ .Response.CamelName }} {{ if .IsResponseByteStream -}}
-    io.ReadCloser
```

Comment:
We are removing a special case for byte stream where we were passing just a field to the cli
```
func (a *billableUsageImpl) Download(ctx context.Context, request DownloadRequest) (*DownloadResponse, error) {
	var downloadResponse io.ReadCloser ->  var downloadResponse DownloadResponse
	path := fmt.Sprintf("/api/2.0/accounts/%v/usage/download", a.client.ConfiguredAccountID())
	headers := make(map[string]string)
	headers["Accept"] = "text/plain"
	err := a.client.Do(ctx, http.MethodGet, path, headers, request, &downloadResponse)
	return &DownloadResponse{Contents: downloadResponse}, err -> return downloadResponse
}
```